sklearn/preprocessing/_polynomial.py

```diff
@@ -55,12 +56,16 @@ class PolynomialFeatures(TransformerMixin, BaseEstimator):
 
     Attributes
     ----------
-    powers_ : ndarray of shape (n_output_features, n_input_features)
+    powers_ : ndarray of shape (n_output_features, n_features_in_)
```

Comment:
I assume that we need to do the following. Otherwise sphinx search for an hyperlink.

```suggestion
    powers_ : ndarray of shape (n_output_features, `n_features_in_`)
```